CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file.
 
+## ## [v0.1.10] Unreleased
```

Comment:
Can we remove the extra ## - only need one of them
Would also remove the [v0.1.10] - what if we were to go to 0.2.0 - even though we do not use semantic versioning and not sure we will ever

Reply:
removed.